src/logic/token-storage.ts

```diff
@@ -0,0 +1,11 @@
+import { TokenBearer } from '../types/types';
+
+export function storeTokenBearer(token: TokenBearer) {
```

Comment:
nit: if we use a record for credentials-storage, we should use a Record here as well.